### PR TITLE
[candi] Rename vendored containerd.service into containerd-deckhouse.service

### PR DESCRIPTION
**Pull request by mvasl**
## Description

Systemd's `containerd.service` shipped with deckhouse is now installed and used as `containerd-deckhouse.service` to avoid corruption of `containerd` packages in OS repos.

Also, a new testing tool is implemented to validate Astra Linux SE security compliance.

## Why do we need it, and what problem does it solve?

This is necessary to comply with Astra Linux SE security requirements.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: chore
summary: Containerd systemd unit that ships with Deckhouse renamed to `containerd-deckhouse.service`.
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/eks'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/static'...

**Review by z9r5 — DISMISSED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/static'...

File: tools/astra_integrity_check/main.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	"flag"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+var gostsumsPath *string
+
+const resultXMLPath = "/tmp/astra-int-check-report.xml"
```
Comment (ViktorKram) [inline tools/astra_integrity_check/main.go:29]: Подскажи, пожалуйста, по кодстайлу константы не идут выше всех по объявлению?

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Review by z9r5 — APPROVED** [review]

**Review by nabokihms — APPROVED** [review]

**Pull request merged**